sycl/test-e2e/Matrix/Legacy/joint_matrix_query_default.cpp

```diff
@@ -7,8 +7,8 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix
 
-// RUN: %clangxx -fsycl %s -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
+// RUN: %{run} %t.out
```

Comment:
GPU was excluded for some reason, now it is not. @dkhaldi , should this test run on GPU?

Reply:
@dkhaldi , what about this one?